app/models/post.rb

```diff
@@ -1013,7 +1013,8 @@ def link_post_uploads(fragments: nil)
       upload ||= Upload.get_from_url(src)
 
       # Link any video thumbnails
-      if upload.present? && (FileHelper.supported_video.include? upload.extension&.downcase)
+      if SiteSetting.video_thumbnails_enabled && upload.present? &&
+           (FileHelper.supported_video.include? upload.extension&.downcase)
```

Comment:
This is a bit more readable:

```suggestion
      if SiteSetting.video_thumbnails_enabled && upload.present? &&
           FileHelper.supported_video.include?(upload.extension&.downcase)
```

Reply:
Yes it is. Let's try it!